scanner/go.mod

```diff
@@ -3,22 +3,37 @@ module github.com/stackrox/stackrox/scanner/v4
 go 1.20
 
 require (
+	github.com/gogo/protobuf v1.3.2
```

Comment:
This is a deprecated fork of a deprecated package. I think we should try to use `google.golang.org/protobuf` instead. I haven't tried it, but I think you can replace `types.Empty` with this: https://pkg.go.dev/google.golang.org/protobuf@v1.30.0/types/known/emptypb

Reply:
2eeb86c674..138832e140

Reply:
The generated protobuf is picking up gogo. If I move to `emptypb` the interface won't match and I won't be able to register my service. This should be dealt with separately.

Reply:
The generated Scanner code is in the main module which requires gogo proto, so I guess we are stuck with it :(